spec/BowlingSpec.js

```diff
@@ -0,0 +1,74 @@
+describe('Bowling', function () {
+
+  var myGlobal
+  var bowling
+  var f
+  var f2
+  var f3
+  var f4
+  var f5
+  var f6
+  var f7
+  var f8
+  var f9
+  var f10
+
+  beforeEach(function() {
+    f2 = new Frame (3, 4)
+    f3 = new Frame (5, 5)
+
+    bowling = new Bowling()
+  })
+
+  it('should add to the list of frames', function () {
+    bowling.addFrame(f)
+    expect(bowling.frames).toContain(f)
+  })
+  it('should only add max 10 frames', function () {
+    var f = new Frame ('frame2')
```

Comment:
This is a good test, but could it be more general? i.e. instead of testing that we can't add more than 10 frames, could we simply test that we couldn't add more than X frames, where X could be 5, 10, 20, 100? 

Obviously this would require a change in the code to allow the max number of frames to be configurable - but whenever we can move from a specific to a general case our code becomes more resilient to change.